### PR TITLE
Clarify the jsontests branch

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ and this project adheres to [Semantic Versioning].
 
 This release introduces complete floating-point support.
 
-With that in place, Fizzy passes all(*) of the [official test suite (spectest 1.0)]:
+With that in place, Fizzy passes all(*) of the [official test suite (spectest 1.0)](https://github.com/wasmx/wasm-spec/releases/tag/w3c-1.0-jsontests-20200813):
```
Comment (axic) [inline CHANGELOG.md:12]: Should these link to `releases/tag/w3c-...` or `tree/w3c-..` ?

File: test/spectests/README.md [hunk 2]
```diff
@@ -33,7 +33,10 @@ In order to prepare the tests, run the following command for each file:
 $ wast2json <file.wast> -o <file.json>
 ```
 
-For ease of use, we have placed the JSON files of the [spec tests] v1.1 release here: https://github.com/wasmx/wasm-spec/tree/vanilla-json
+For ease of use, we have placed the JSON files of the spec tests [w3c-v1.0 branch] here:
+- [Used before Fizzy 0.4](https://github.com/wasmx/wasm-spec/releases/tag/w3c-1.0-jsontests-20200313)
```
Comment (axic) [inline test/spectests/README.md:37]: Do we need to mention the branch `vanilla-json` here too?

**Review by chfast — APPROVED** [review]

File: test/spectests/README.md (hunk 2 above)
Comment (chfast) [reply test/spectests/README.md:37]: No.

File: test/spectests/README.md [hunk 3]
```diff
@@ -33,7 +33,10 @@ In order to prepare the tests, run the following command for each file:
...
+- [Used after Fizzy 0.4](https://github.com/wasmx/wasm-spec/releases/tag/w3c-1.0-jsontests-20200813)
```
Comment (gumb0) [inline test/spectests/README.md:38]:
```suggestion
- [Used in Fizzy 0.4 and above](https://github.com/wasmx/wasm-spec/releases/tag/w3c-1.0-jsontests-20200813)
```

**Review by gumb0 — APPROVED** [review]

File: test/spectests/README.md (hunk 3 above)
Comment (axic) [reply test/spectests/README.md]: Fixed.

**Review by gumb0 — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (chfast) [reply CHANGELOG.md:12]: Maybe `https://github.com/wasmx/wasm-spec/tree/w3c-1.0-jsontests-20200813/test/core/json`?

**Review by chfast — APPROVED** [review]

**Pull request merged**